tool/site/client/src/services/zip-files.ts

```diff
@@ -0,0 +1,19 @@
+import JSZip from "jszip"
+import FileSaver from "file-saver"
+
+interface ZipFileItem {
+  fileName: string
+  item: string
+}
+
+export const zip = async (zipFileName: string, itemsToBeZipped: Array<ZipFileItem>): Promise<void> => {
+  const zip = JSZip()
+
+  itemsToBeZipped.forEach(item => {
+    zip.file(item.fileName, item.item)
+  })
+
+  zip.generateAsync({type: "blob"}).then(zipFile => {
```

Comment:
not super important but we have generally used `async await` in this project rather than the `.then` style of handling promises, could we change this to be consistant, something like:

```
const zipFile = await zip.generateAsync({type: "blob"})

return FileSaver.saveAs(zipFile, `${zipFileName}.zip`)
```

Reply:
true, too easy to copy paste